core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -188,35 +201,41 @@ object Battle {
         damageToAttacker -= attacker.getHealth() // ... but from here on they are accurate
         damageToDefender -= defender.getHealth()
 
-        plunderFromDamage(attacker, defender, damageToDefender)
-        plunderFromDamage(defender, attacker, damageToAttacker)
+        plunderFromDamage(attacker, defender, damageToDefender, "Earn []% of the damage done to [] units as []")
+        plunderFromDamage(defender, attacker, damageToAttacker, "Earn []% of the damage done while defending against [] units as []")
```

Comment:
I'm not sure why you added this. 
Does this appear in the base game?

Reply:
Since the answer is no, this will go including side effects. Why? Somewhere I mentioned it - someone's idea to allow modding 'plunder while defending'.